README.md

```diff
@@ -2,8 +2,32 @@
 
 The source for the Ember.js guides.
 
+
 ### Contributing
 
+The Ember Guides are maintained and updated by an all-volunteer group of Ember community members. We'd love to have you join our efforts! 
+
+Please note that no attempt is made to update content, layout, or styles for older versions of the Guides. They are considered static and immutable, as it is too difficult to maintain content for every version ever released. Issues will only be fixed for future releases.
+
+## Fixing problems
+
+If you find a problem on a particular page in the Guides, the most helpful thing you can do is open a pull request. If you're not sure how to fix it, open an issue.
+
+## Contributing solicited content
+
+We try to make it easy for people to contribute to the Guides by tagging issues with _Help Wanted_ when appropriate. The best way to get started contributing content is to pick up one of these issues.
+
+## Contributing unsolicited content
+
+If you'd like to contribute content that you think is missing, please start by checking the issues page. There may already be a plan to add this content! If not, open an issue yourself so that you can get feedback before you start writing. Our core contributors may ask you to start off by writing a blog post on your topic before turning it into a pull request on the Guides. This helps us keep the Guides consistent and streamlined.
```

Comment:
I don't know how to phrase it, but I'd put more emphasis on generating/consulting community content than opening a new issue.